rest/rest-cli/src/test/java/functionaltests/TagCommandsFunctTest.java

```diff
@@ -24,14 +24,16 @@ public static void beforeClass() throws Exception {
 
     @Before
     public void setUp() throws Exception {
-        super.setUp();
-        if(jobId == null) {
-            cleanScheduler();
-
-            //submit a job with a loop and out and err outputs
-            System.out.println("submit a job with loop, out and err outputs");
-            jobId = submitJob("flow_loop_out.xml", JobStatus.FINISHED);
-            System.out.println("Job " + jobId + " finished");
+        synchronized (TagCommandsFunctTest.class) {
```

Comment:
I didn't want to touch it ... didn't manage really to understand the issue as you know.